zeppelin-zengine/src/main/java/org/apache/zeppelin/interpreter/launcher/SparkInterpreterLauncher.java

```diff
@@ -106,7 +106,7 @@ public SparkInterpreterLauncher(ZeppelinConfiguration zConf, RecoveryStorage rec
               " to false if you want to use other modes.");
     }
 
-    StringBuilder sparkConfBuilder = new StringBuilder();
+    List<String> sparkConfList = new ArrayList<>();
```

Comment:
A StringJoiner is more elegant

https://docs.oracle.com/javase/8/docs/api/java/util/StringJoiner.html

Reply:
Thanks for the review, PR is updated, `SparkInterpreterLauncher` & `FlinkInterpreterLauncher` both use StringJoiner now.